### ISSUE TITLE
Form label missing

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#219]
A form &lt;input&gt;, &lt;select&gt;, or &lt;textarea&gt; does not have a corresponding label. (Note: Labels are not required for image, submit, reset, button, or hidden form element types.)

Properly associate a form label with the input element.
